CHANGELOG.md

```diff
@@ -3,41 +3,57 @@
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+and this project adheres to
+[Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-### [Next]
+## [Unreleased]
+
+## [0.4.2]
+
+### Changed
+
+- update itertools to 0.13.0
```

Comment:
Also `positional_derive`'s dependency on `syn` seems to have been bumped to v2.

Reply:
I felt there had been other changes, thanks!